safe/gis/vector/summary_4_exposure_summary_table.py

```diff
@@ -44,7 +47,8 @@
 
 
 @profile
-def exposure_summary_table(aggregate_hazard, callback=None):
+def exposure_summary_table(
+        aggregate_hazard, exposure_summary=None, callback=None):
```

Comment:
You are breaking the waterfall! ;-)
Not possible to summarize first in the aggregate_hazard layer ?

Sorry for you to have to update these files summary_*.py, these are a little bit messy.